docs/error-reporting-usage.rst

```diff
@@ -76,6 +76,19 @@ be used by Stackdriver Error Reporting to help group exceptions.
    >>> except Exception:
    >>>     client.report_exception(http_context=http_context, user=user))
 
+An automatic helper to build the HTTP Context from a Flask (Werkzeug) request
+object is provided.
+
+ .. code-block:: python
+
+    >>> @app.errorhandler(Exception)
```

Comment:
Pretty please don't use `Exception`, `GoogleCloudError` could be the right choice?